volatility3/cli/__init__.py

```diff
@@ -88,7 +88,7 @@ def __call__(self, progress: Union[int, float], description: str = None):
 class CommandLine:
     """Constructs a command-line interface object for users to run plugins."""
 
-    CLI_NAME = "volatility"
+    CLI_NAME = os.path.basename(sys.argv[0])  # vol or volatility
```

Comment:
This should be fine (there's possible unintended consequences, but it gets vol and volshell more in alignment so I don't think we need to worry about people playing games with their own installations.